flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/sink/InitContextInitializationContextAdapter.java

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-package org.apache.flink.api.connector.sink;
+package org.apache.flink.streaming.runtime.operators.sink;
 
 import org.apache.flink.api.common.serialization.SerializationSchema;
+import org.apache.flink.api.connector.sink.Sink;
```

Comment:
We accidentally introduced this class when developing the KafkaSink but it was always meant to be public. The additional import it required because before the class was in the same package as the `Sink` so no import was required. Now the class is close to the `SinkWriterOperator` that is now the only place where it is used.